pennylane/ops/qubit/qchem_ops.py

```diff
@@ -77,6 +84,9 @@ def circuit(phi):
     num_params = 1
     """int: Number of trainable parameters that the operator depends on."""
 
+    ndim_params = (0,)
+    """tuple[int]: Number of dimensions per trainable parameter that the operator depends on."""
```

Comment:
`ndim_params` contains one integer per parameter of the operation. Each integer describes how many dimensions the parameter is expected to have. For most operations, a single scalar is expected, so that `ndim_params=(0,)`. 
A different example would be `QubitStateVector`, which expects a single vector, so `QubitStateVector.ndim_params=(1,)`, or `Rot`, which expects three scalars, so that `Rot.ndim_params=(0, 0, 0)`.
That being said, the docstring should be improved, then. Would you have an idea of what part is unclear in particular, or a phrasing that better conveys the above? :)

Reply:
It is clear now, thanks.